mephisto/abstractions/blueprints/static_html_task/source/dev/app.jsx

```diff
@@ -110,11 +111,19 @@ function MainApp() {
 
 function SubmitFrame({ children, onSubmit, currentTask }) {
   const [submitting, setSubmitting] = React.useState(false);
+  const [
+    isSubmitButtonHidden,
+    setIsSubmitButtonHidden,
+  ] = useMephistoGlobalConfig(
+    "HIDE_SUBMIT_BUTTON",
+    false,
+    (val) => typeof val === "boolean"
+  );
 
   React.useEffect(() => {
     // Reset submitting when switching from onboarding
     setSubmitting(false);
-  }, [currentTask]);
+  }, []);
```

Comment:
what was the rationale behind removing this?

Reply:
that's a mistake, I don't even know how that got updated.